task/fbc-validation/0.1/README.md

```diff
@@ -0,0 +1,24 @@
+# fbc-validation task
+
+## Description:
+The fbc-validation task will ensure FBC (File based catalog) components uniquely linted to ensure they're properly
+constructed as part of the build pipeline. To validate the image in build pipeline, Skopeo is used to extract
+information and then contents are checked using the OpenShift Operator Framework.
+
+## Params:
+
+| name         | description                     |
+|--------------|---------------------------------|
+| image-digest | Image digest.                   |
+| image-url    | The fully qualified image name. |
+
+## Params:
+
+| name               | description        |
+|--------------------|--------------------|
+| HACBS_TEST_OUTPUT  | Test output.       |
+
+## Additional links:
+https://olm.operatorframework.io/docs/reference/file-based-catalogs/
+https://github.com/containers/skopeo
+https://docs.openshift.com/container-platform/4.12/cli_reference/opm/cli-opm-install.html
```

Comment:
This isn't getting rendered properly, so I guess these links need to be a part of the bullet point list.
```suggestion
* https://olm.operatorframework.io/docs/reference/file-based-catalogs/
* https://github.com/containers/skopeo
* https://docs.openshift.com/container-platform/4.12/cli_reference/opm/cli-opm-install.html
```

Reply:
Nice catch, it looks like the markdown will concatenate multiple links if they are only separated by new lines in the md. I ended up using <br> tags for line breaks so that all the singular links wouldn't need updating to keep consistent but I'm not a huge fan of either solution to be honest.